Makefile

```diff
@@ -19,13 +26,15 @@ release: test tag_latest
 	docker push $(NAME)
 	@echo "*** Don't forget to create a tag by creating an official GitHub release."
 
+ssh: SSH_COMMAND?=
+ssh: SSH_IDENTITY_FILE?=image/services/sshd/keys/insecure_key
 ssh:
-	chmod 600 image/services/sshd/keys/insecure_key
-	@ID=$$(docker ps | grep -F "$(NAME):$(VERSION)" | awk '{ print $$1 }') && \
+	chmod 600 ${SSH_IDENTITY_FILE}
+	ID=$$(docker ps | grep -F "$(NAME):$(VERSION)" | awk '{ print $$1 }') && \
 		if test "$$ID" = ""; then echo "Container is not running."; exit 1; fi && \
-		IP=$$(docker inspect $$ID | grep IPAddr | sed 's/.*: "//; s/".*//') && \
```

Comment:
Probably docker version dependent, but the `sed` substitution wasn't giving the expected output, so I've found out how the do the same with docker itself.